modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -506,6 +507,39 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
         assertEquals(50, result.assets[DEFAULT_ASSET_ID]?.value?.cast<AssetValue.Quantity>()?.u32)
     }
 
+    @Test
+    @WithIroha([AliceHas100XorAndPermissionToBurn::class])
+    @Feature("Assets")
+    @Story("Account burns an asset")
+    @Permission("no_permission_required")
+    @SdkTestId("burn_asset_for_account_in_same_domain")
+    fun `multi signature transaction`(): Unit = runBlocking {
+        val newBobKeyPair = generateKeyPair()
+        val newBobPublicKey = newBobKeyPair.public.toIrohaPublicKey()
+
+        client.tx(BOB_ACCOUNT_ID, BOB_KEYPAIR) {
+            mintSignatureCheckCondition(
+                BOB_ACCOUNT_ID,
+                SignatureCheckCondition.AllAccountSignaturesAnd(listOf(newBobPublicKey)),
```

Comment:
either:
1. don't specify additional signature here (use empty list) and mint a signature for the account
2. specify the signature here (as you did) and don't mint public key for the account

Reply:
I would recommend 1st approach which means:
```suggestion
                SignatureCheckCondition.AllAccountSignaturesAnd(listOf()),
```

Reply:
I've got a question related to option 2. Why isn't minting of the public key not necessary in that case?

I guess it's better to specify the exact key expected and verify other key won't work.

Reply:
> I've got a question related to option 2. Why isn't minting of the public key not necessary in that case?
> 
> I guess it's better to specify the exact key expected and verify other key won't work.

because look at the name `AllAccountSignaturesAnd`. It means that, for a transaction to be valid, we require all signatures that are minted for that account and (optionally) some additional signatures that are not owned by that account